web-portal/pages/admin-venue-edit/index.vue

```diff
@@ -0,0 +1,139 @@
+
+      <ii-pagination v-show="!isFetching" :items="selectedVenueList">
+        <template slot-scope="page">
+          <venue-card
+            v-for="venue in page"
+            :venue="venue"
+            :key="venue.id"
+          />
+        </template>
+      </ii-pagination>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { FETCH_ACTIVE_VENUES, FETCH_DELETED_VENUES } from '../../store/venues'
+  import VenueCard from '../../components/admin-venue-edit/VenueCard'
+  import VenueSpinner from '../../components/admin-venue-edit/VenueSpinner'
+  import Pagination from '../../components/Pagination'
+
+  const ACTIVE_VENUE_SELECTION = 'Active Venues'
+
+  const sortMethod = (venueA, venueB) => venueA.name > venueB.name ? 1 : -1
+
+  export default {
+    components: { VenueSpinner, VenueCard, 'ii-pagination': Pagination },
+    layout: 'admin',
+    middleware: 'auth',
+    fetch: function () {
+      if (this.selectedList === ACTIVE_VENUE_SELECTION) {
+        return this.$store.dispatch(FETCH_ACTIVE_VENUES)
+      } else {
+        return this.$store.dispatch(FETCH_DELETED_VENUES)
+      }
+    },
+
+    methods: {
+      onFilter() {
+        this.$fetch()
+      }
+    },
+
+    computed: {
+      isActiveVenuesFetching: function () {
+        return this.$store.state.venues.getActiveVenuesQuery.isFetching
+      },
+
+      isDeletedVenuesFetching: function () {
+        return this.$store.state.venues.getDeletedVenuesQuery.isFetching
+      },
+
+      selectedVenueList: function () {
+        const selectedList = this.isShowingActive
+          ? this.activeVenues
+          : this.deletedVenues
+
+        if (this.isSearchEntered) {
+          return selectedList.filter(value => (
+            value.name.toLowerCase().indexOf(this.searchByNameValue.toLocaleLowerCase()) >= 0))
+        } else {
+          return selectedList
+        }
+      },
+
+      isSearchEntered: function () {
+        return this.searchByNameValue.trim().length > 0
+      },
+
+      activeVenues: function () {
+        return [...this.$store.state.venues.getActiveVenuesQuery.data].sort(sortMethod)
+      },
+
+      deletedVenues: function () {
+        return [...this.$store.state.venues.getDeletedVenuesQuery.data].sort(sortMethod)
+      },
+
+      errorFetchingActiveVenues: function () {
+        return this.$store.state.venues.getActiveVenuesQuery.error
+      },
+
+      errorFetchingDeletedVenues: function () {
+        return this.$store.state.venues.getDeletedVenuesQuery.error
+      },
+
+      isShowingActive: function () {
+        return this.selectedList === 'Active Venues'
+      },
+
+      isFetching: function () {
+        return this.$store.state.venues.getDeletedVenuesQuery.isFetching ||
+          this.$store.state.venues.getActiveVenuesQuery.isFetching
+      }
+    },
+
+    data() {
+      return {
+        selectedList: 'Active Venues',
+        searchByNameValue: ''
+      }
+    }
+  }
+</script>
+
+<style>
```

Comment:
For reference (because we're talking about it right now on Zoom) `<style scoped>` would scope this CSS to this component.

Reply:
good idea, applying change